src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/BaseAnalysisController.java

```diff
@@ -693,7 +686,22 @@ public void getAllAnalysisFiles(
         return new JsonResult<>(NO_ERROR, Boolean.TRUE);
     }
 
-    protected List<MultipartFile> getEntityFiles(DataReferenceDTO entityReference, DataNode dataNode, CommonAnalysisType entityType)
+
+    protected byte[] readResource(final String path) throws IOException {
```

Comment:
put protected methods at the end of the class.

Reply:
done